jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/DockerFileUtil.java

```diff
@@ -103,15 +102,36 @@ private DockerFileUtil() {}
      *
      * @param dockerFile docker file to interpolate.
      * @param properties properties to interpolate in the provided dockerFile.
+     * @param filter filter for parsing properties from Dockerfile
+     * @return The interpolated contents of the file.
+     * @throws IOException if there's a problem while performin IO operations.
+     */
+    public static String interpolate(File dockerFile, Properties properties, String filter) throws IOException {
+        StringBuilder ret = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(dockerFile))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                ret.append(JKubeDockerfileInterpolator.interpolate(line, properties, filter)).append(System.lineSeparator());
+            }
+        }
+        return ret.toString();
+    }
+
+     /**
+     * Interpolate a docker file with the given properties and filter
+     *
+     * @param dockerFile docker file to interpolate.
+     * @param properties properties to interpolate in the provided dockerFile.
+      * @param customInterpolationParams additional delimiters for interpolation
      * @return The interpolated contents of the file.
      * @throws IOException if there's a problem while performin IO operations.
      */
-    public static String interpolate(File dockerFile, Properties properties) throws IOException {
+    public static String interpolate(File dockerFile, Properties properties, Map<String, String> customInterpolationParams) throws IOException {
         StringBuilder ret = new StringBuilder();
         try (BufferedReader reader = new BufferedReader(new FileReader(dockerFile))) {
             String line;
             while ((line = reader.readLine()) != null) {
-                ret.append(JKubeDockerfileInterpolator.interpolate(line, properties)).append(System.lineSeparator());
+                ret.append(JKubeDockerfileInterpolator.interpolate(line, properties, customInterpolationParams)).append(System.lineSeparator());
```

Comment:
No longer used, since we have `public static String interpolate(File dockerFile, Properties properties, String filter) throws IOException {, please remove (test too)